LondonBritishLibrary/orient/BLorient797.xml

```diff
@@ -124,13 +78,110 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                            </objectDesc>
+                            <handDesc>
+                                <handNote script="Ethiopic" xml:id="h1" corresp="">
+                                    <desc>Written in a fine hand.</desc>
+                                    <date notBefore="1700" notAfter="1750"/>
+                                </handNote>
+                                <handNote xml:id="h2" script="Ethiopic" corresp="">
+                                  <date notBefore="1700" notAfter="1750"/>
+                                </handNote>
+                            </handDesc>
+                            <additions>
+                                <list>
+
+                                  <item xml:id="a1">
+                                    <locus target="#3r"></locus>
+                                    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+                                        <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                                    <q xml:lang="gez">ዲድስቅልያ፡ ዘ<placeName ref="INS0101MadhaneAlam"> ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName>
+                              </q>
+                                  </item>
+
+                                  <item xml:id="a2">
+                                    <locus from="109r"></locus>
+                                    <desc type="GuestText">
+                                      <title ref="LIT1994Mazgab"></title>
+                                    </desc>
+                                  <q xml:lang="gez"></q>
+                                  </item>
+                                  <item xml:id="a3">
+                                    <locus from="122r"></locus>
+                                    <desc>
+                                      <title ref="LIT2293Sawase" xml:lang="en">Explanation of words,
+                                      chiefly Biblical in Amharic.</title>
+                                    </desc>
+                                    <q xml:lang="am">
+                                      <seg part="I">
+                                        ንጽሕፍ፡ ነገረ፡ ሰዋስው፡ ዘሐዲስ፡ ወዘብሉይ። ሰዋስው፡ ዘ፡ መሽጋገርያ፡ ወዓዲ፡ ዘ፡ መሰላል፡ ዲበ፡ ዘ፡
+                                        በላ፡ እንዘ፡ ዘ በ፫፡ ሲሰስ፡ እስመ፡ ዘ፡ በ፪፡ ኮን፡ ስለ፡ እመ፡ ሶበ፡ ዘ፡ ፪፡ በቢቡብ፡ እንከ፡ ዘ፡ እንግድያ፡
+                                        እፎ፡ ዘ፡ በ፪፡ እንዴት፡ ምን፡ እምዝ፡ ዘ፡ በ፪፡ ከዝያ፡ እንግዴኅ፡ እንተ፡ እለ፡ ዘ፡ የ፡ ይእዜ፡ ዘ፡ በ፫፡
+                                        አኁን፡ ዛሬም፡ እንግዴኅ፤
+                                      </seg>
+                                    </q>
+                                  </item>
+                                </list>
+                            </additions>
+
+
+                        </physDesc>
+
+                        <history>
+                            <origin>
+                                <origDate notBefore="1400" notAfter="1450" evidence="lettering"/>
+                            </origin>
```

Comment:
Is the date here correct?

Reply:
Sorry, Denis, I think you have commented not on the last version of the file. In the last version the date is 1700-1750